x-pack/plugins/endpoint/public/applications/endpoint/lib/saga.test.ts

```diff
@@ -7,13 +7,12 @@
 import { createSagaMiddleware, SagaContext, SagaMiddleware } from './index';
 import { applyMiddleware, createStore, Reducer, Store } from 'redux';
 
-// Failing: https://github.com/elastic/kibana/issues/55464 https://github.com/elastic/kibana/issues/55465
-describe.skip('saga', () => {
+describe('saga', () => {
   const INCREMENT_COUNTER = 'INCREMENT';
   const DELAYED_INCREMENT_COUNTER = 'DELAYED INCREMENT COUNTER';
   const STOP_SAGA_PROCESSING = 'BREAK ASYNC ITERATOR';
 
-  const sleep = (ms = 10) => new Promise(resolve => setTimeout(resolve, ms));
+  const sleep = (ms = 100) => new Promise(resolve => setTimeout(resolve, ms));
```

Comment:
I'm worried about this continuing to be a problem down the road.  I assume `10` was intermittent depending on resources during CI.  `100` would certainly make it occur less often, but kibana is a constantly growing application.  Just weighing the values of this test vs. the risk of intermittent failures.  What are your thoughts?

Reply:
Yeah, I looked at this for a while yesterday trying to see if there was another way. Because this lib is the core for us doing side-effects, I think there is high value in ensuring that it behaves as expected.  I searched around other x-pack plugins and I do see `100` (or in some cases higher values) being used. Because there are not allot of tests here, I can also increase the `ms` to a higher number in order to try to avoid this type of failure again.

Reply:
got it, thanks for the reply and taking a look around at the other plugins.